CommandLine/testing/SimpleTests/file_input_output.sog/create.edl

```diff
@@ -198,6 +198,7 @@ gtest_expect_eq(filename_drive("C/John/Doe/Smoe.txt"),"");
 gtest_expect_eq(filename_drive("/c/John/Doe/Smoe.txt"),"");
 gtest_expect_eq(filename_drive("Smoe.txt"),"");
 gtest_expect_eq(filename_ext("C:/John/Doe/Smoe.txt"),".txt");
+gtest_expect_eq(filename_ext("C:/John/Doe/Smoe.gmx/datafiles/Makefile"),"");

```

Comment:
That's good, you can also test this too since these are all very cheap:
```cpp
gtest_expect_eq(filename_ext("C:/John/Doe/Smoe.gmx/datafiles/Makefile.txt"),"");
```

I was waiting to see what you were going to test, I hadn't expected that's what you were testing, but ok seems reasonable.